docs/application/vstools/tools/emulator-manager.md

```diff
@@ -1,152 +1,163 @@
 # Emulator Manager
 
-You can use emulators to run your application in a virtual environment.
+After you have built your application, you need to test the application to ensure quality. To test and debug your application, you can configure a virtual environment before deploying your application. You can use emulators to run and test your application in a virtual environment. Tizen provides an emulator, Tizen Emulator Manager (Emulator Manager). It helps you to test your application in an environment similar to a real device.
```

Comment:
"After you have built your application, you need to test the application to ensure quality. To test and debug your application, you can configure a virtual environment before deploying your application."
These sentences are not good as introduction. change it.